python-avd/pyavd/_eos_designs/structured_config/__init__.py

```diff
@@ -41,7 +42,7 @@
     # Metadata must be after anything else that can generate structured config, since CV tags can consume from structured config.
     AvdStructuredConfigMetadata,
     # The Custom Structured Configuration module must be rendered last,
-    # since it parses all supported object looking for `struct_cfg`.
+    # since it applies the structured config snips gathered by the other generators.
```

Comment:
```suggestion
    # since it strips empties from all previously generated structured config and then
    # applies the structured config snips gathered by the other generators.
```